README.md

```diff
@@ -40,132 +39,118 @@ Create a new .env file for the deployment based on the dotenv file included.
 cp dotenv .env
 ```
 
-The only variable required is the domain name that will be used, since by default, the app will be exposed as gl.DOMAIN_NAME for greenlight and kc.DOMAIN_NAME for keycloak.
+Most required variables are pre-set by default, the ones that must be set before starting are:
+
 ```
-DOMAIN_NAME=
+POSTGRES_PASSWORD= # Postgres user password (DO NOT SET THIS).
+KEYCLOAK_PASSWORD= # Keycloak admin password (DO NOT SET THIS).
 ```
 
-Also, when using the `init-letsencrypt.sh` script, you should add the email.
-
+It's recommonded to generate secure random credentials such as the **POSTGRES_PASSWORD** and the **KEYCLOAK_PASSWORD**.
+They can be generated by running:
 ```
-LETSENCRYPT_EMAIL=
+psql_pwd="$(openssl rand -hex 40)";kc_pwd="$(openssl rand -hex 40)"
+sed -i "s/POSTGRES_PASSWORD=.*/POSTGRES_PASSWORD=$psql_pwd/" .env
+sed -i "s/KEYCLOAK_PASSWORD=.*/KEYCLOAK_PASSWORD=$kc_pwd/" .env
+sed -i "s/DATABASE_URL=.*/DATABASE_URL=postgres:\/\/postgres:$psql_pwd@postgres:5432/" data/greenlight/.env
+unset psql_pwd kc_pwd;
 ```
+> Note: Whatever is in **$KEYCLOAK_PASSWORD** will be your Keycloak admin account password.
 
-## For using a SSL certificate signed by Let’s Encrypt, generate the certificates.
-
-### Manual
-
+We also need to set in **.env**:
 ```
-certbot certonly --manual -d gl.<DOMAIN_NAME> --agree-tos --no-bootstrap --manual-public-ip-logging-ok --preferred-challenges=dns --email your@email.com --server https://acme-v02.api.letsencrypt.org/directory
+DOMAIN_NAME= # The domain name of the applications, along with a hostname it will form the application FQDN (Fully Qualified Domain Name).
+GL_HOSTNAME= # Greenlight hostname, the Greenlight FQDN will be $GL_HOSTNAME.$DOMAIN_NAME.
+KC_HOSTNAME= # Keycloak hostname, the Keycloak FQDN will be $KC_HOSTNAME.$DOMAIN_NAME.
 ```
 
-#### Validate the Let’s Encrypt challenge
-
-Respond yes to sharing the Email and IP
-
-Take the string given as a challenge and create a TXT record on your DNS (see the prerequisites). You should set up the Challenge as indicated.
-
+Example:
+If your Greenlight FQDN was **gl.demo.bigbluebutton.org** and Keycloak's was  **kc.demo.bigbluebutton.org**.
+You'd go to the **.env** and set:
 ```
-Performing the following challenges:
```

Comment:
Removed in favor of the script,
Those who are willing to use a certificate (a wildcard, not issued by letsencrypt) will have the docs on how to use their already owned cert.